lib/portage/dbapi/vartree.py

```diff
@@ -4710,83 +4691,91 @@ def onerror(e):
         rofilesystems = ro_checker(dirs_ro)
 
         if rofilesystems:
-            msg = _(
-                "One or more files installed to this package are "
-                "set to be installed to read-only filesystems. "
-                "Please mount the following filesystems as read-write "
-                "and retry."
-            )
-            msg = textwrap.wrap(msg, 70)
-            msg.append("")
-            for f in rofilesystems:
-                msg.append("\t%s" % f)
-            msg.append("")
+            msg = [
+                *textwrap.wrap(
+                    _(
+                        "One or more files installed to this package are "
+                        "set to be installed to read-only filesystems. "
+                        "Please mount the following filesystems as read-write "
+                        "and retry."
+                    ),
+                    70,
+                ),
+                "",
+                *(f"\t{f}" for f in rofilesystems),
+                "",
+            ]
```

Comment:
Same here.

Reply:
And I'm still not sure what you're requesting.